tool/python/singa/layer.py

```diff
@@ -228,21 +228,20 @@ def __init__(self, size=0, **kwargs):
 
 class Activation(Layer):
 
-    def __init__(self, activation='stanh', topk=1):
+    def __init__(self, activation='stanh'):
         '''
         required
-          activation = (string)
-        optional
-          topk       = (int)  // the number of results
+          activation = (string) // relu, sigmoid, tanh, stanh, softmax.
         '''
 
+        if activation == 'tanh':
+          print 'Warning: Tanh layer is not supported for CPU'
+
         self.name = activation
-        if activation == 'tanh': activation = 'stanh' # <-- better way to set?
         self.layer_type = enumLayerType(activation)
```

Comment:
Do you process cpu tanh and cpu stanh in the same way?
Currently, tanh can be implemented by creating an ActivationLayer and setting its type to be TANH. STANH is implemented by creating a STanHLayer.
I think line 241 only gets the layer type. line 242 passes it to create the Layer object. 
Where do you set the activation type for tanh?